### ISSUE TITLE
Cannot find module

### ISSUE THREAD

**Issue by thomaspaulmann** [linear/linear#37]
I tried to use the SDK for a simple CLI tool but got stuck with an internal build error. After following the getting started steps, I got the following error message when building with `tsc`:

```
node_modules/@linear/sdk/dist/_generated_documents.d.ts:1:51 - error TS2307: Cannot find module '@graphql-typed-document-node/core' or its corresponding type declarations.

1 import { TypedDocumentNode as DocumentNode } from "@graphql-typed-document-node/core";
```

I set up a [simple repository](https://github.com/thomaspaulmann/linear-script) with my local setup that hopefully helps to debug the issue.

**Comment by littlemooon** [comment]
@thomaspaulmann Thank you for the reproduction!

This is now fixed in 1.4.4:
https://github.com/linear/linear/tree/%40linear/sdk%401.4.4

And the readme updated in 1.4.5 as we no longer require the peer dependencies:
https://github.com/linear/linear/tree/%40linear/sdk%401.4.5

Here's a fork commit that fixes your repo :)
https://github.com/littlemooon/linear-script/commit/a6183b10ff80f5d17976a8c9d45953c3a0ba28d7

**Comment by thomaspaulmann** [comment]
Thanks for looking into this quickly and great that you removed the peer dependencies. This makes it nicer to deal with in general 👏
